### PR TITLE
Added data-cy label to SecondaryMenuTarget

### LINKED ISSUE #1168: Add data-cy label to SecondaryMenuTarget component
- [x] Add data-cy label to SecondaryMenuTarget component.
- [x] Add label to Dropdown in EmojiOption to add the data-cy label to Dropdown.

### PR DESCRIPTION
**Pull request by deepanshu-rajput-git**
- Fixes #1168

**Description**
 Added data-cy label to SecondaryMenuTarget component.
 Added label to Dropdown in EmojiOption to add the data-cy label to Dropdown.

**Checklist**

- [x] ~~I have made corresponding changes to the documentation.~~
- [x] ~~I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

### CODE REVIEW

**Comment by deepanshu-rajput-git** [comment]
@AbhayVAshokan _a 
@VarunSriram99 _a
@shreyakurian02 _a
Please review this PR

File: src/components/Editor/Menu/Fixed/components/EmojiOption.jsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ const EmojiOption = ({
     dropdownProps={{ classNames: "neeto-editor-fixed-menu__emoji-dropdown" }}
     icon={Smiley}
     isOpen={isActive}
+    label="emoji"
```
Comment (AbhayVAshokan) [inline src/components/Editor/Menu/Fixed/components/EmojiOption.jsx:24]: image

We do not need a label for the Emoji option. The icon is self explanatory.

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@deepanshu-rajput-git _a

**Comment by deepanshu-rajput-git** [comment]
@AbhayVAshokan _a we have fixed the emoji icon label, please review the new commits.

**Review by AbhayVAshokan — APPROVED** [review]

**Pull request merged**